ci-operator/config/red-hat-storage/df-console/red-hat-storage-df-console-main.yaml

```diff
@@ -0,0 +1,69 @@
+base_images:
+  base:
+    name: release
+    namespace: openshift
+    tag: tectonic-console-builder-v23
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: tectonic-console-builder-v23
+images:
+  - dockerfile_path: Dockerfile.ci
```

Comment:
we need to add this file to dfr-console repo... once we have stable images for the plugin...

Reply:
We can define the image build procedures in the `Dockerfile.ci`?